bank.cpp

```diff
@@ -248,8 +290,23 @@ void Bank::transactions()
 
         int amount;
 
+        std::system("clear");
         std::cout << "Enter the amount\n";
-        std::cin >> amount;
+        std::string str_amount;
```

Comment:
It would be better to use integer type instead of std::string

Reply:
ok!